lib/http/2/compressor.rb

```diff
@@ -146,6 +146,9 @@ def process(cmd)
 
         case cmd[:type]
         when :changetablesize
+          if cmd[:value] > @limit
+            fail ProtocolError, 'dynamic table size update exceed limit'
```

Comment:
> A value that exceeds this limit MUST be treated as a decoding error.

WDYT of raising [CompressionError](https://github.com/igrigorik/http-2/blob/master/lib/http/2/error.rb#L16-L19)?

Reply:
Sounds good, updated in the latest commit : )